base/libgit2/repository.jl

```diff
@@ -239,6 +239,69 @@ function peel{T<:GitObject}(::Type{T}, obj::GitObject)
 end
 peel(obj::GitObject) = peel(GitObject, obj)
 
+"""
+    LibGit2.GitDescribeResult(commitish::GitObject; kwarg...)
+
+Produce a `GitDescribeResult` of the `commitish` `GitObject`, which
+contains detailed information about it based on the keyword argument:
+
+  * `options::DescribeOptions=DescribeOptions()`
+
+Equivalent to `git-describe <commitish>`.
```

Comment:
I don't think there should be a hyphen between `git` and `describe`; `git describe` is the CLI AFAICT